spec/controllers/sessions_controller_spec.rb

```diff
@@ -40,4 +40,26 @@
     end
   end
 
+  describe "DELETE #logout" do
+    context "when logout triggered" do
+
+      let(:existing_user) { create(:user) }
+
+      before(:each) do
+        allow(User).to receive(:find_by_email).
+            and_return(existing_user)
+        allow_any_instance_of(User).to receive(:authenticate).
+            and_return(existing_user)
+      end
+
+      it "redirects to homepage" do
+        post :create, params: { session: {email: existing_user.email,
+                                          password: existing_user.password} }
+        delete :destroy
+        expect(response).to redirect_to "/"
+      end
```

Comment:
Might be good to add a test to check that the session variable has been cleared? You should be able to access the `session` variable within the test itself.

Reply:
Good thinking, will add that in now!

Reply:
Although the scope itself isn't part of this issue, **Logout**, I believe we could take the opportunity to update the other `redirect_to` calls in the code.